src/harness/reference_models/iap/test_data/fss_0.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "incumbent/ibfs/0",
+  "type": "FSS",
+  "deploymentParam":
+  [
+    {
+      "installationParam": {
+        "latitude": 39.58875,
+        "longitude":  -99.27246,
+        "elevation": 100,
+        "height": 26.5,
+        "heightType": "AGL",
+        "antennaGain": 16,
+        "antennaBeamwidth": 30,
+        "antennaAzimuth": 180,
+        "antennaDowntilt": 5,
+        "antennaElevationAngle":39
+      },
+      "operationParam": {
+        "operationFrequencyRange": {
+          "lowFrequency": 3600000000,
+          "highFrequency": 4200000000
+        }
+      },
+      "weight1": 0.5,
+      "weight2": 0.5,
```

Comment:
weight1 and weight2 are not part of the fss registration info

Reply:
removed